bundles/storage/org.eclipse.smarthome.storage.mapdb/src/main/java/org/eclipse/smarthome/storage/mapdb/internal/MapDbStorage.java

```diff
@@ -9,10 +9,13 @@
 
 import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
 import java.util.HashSet;
 import java.util.Map;
 
 import org.eclipse.jdt.annotation.NonNull;
```

Comment:
There is still @NonNull in use in this class, which should be superseded by @NonNullByDefault.